### PR TITLE
Add pytorch best practices page

### PR DESCRIPTION
**Pull request by ericdatakelly**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/how-tos/pytorch-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+So just go to conda-store and create a new environment by adding `pytorch`, `torchvision`, `torchaudio`, and `pytorch-cuda=11.7` to the list of requested packages and adding `pytorch` and `nvidia` to the list of channels.
+
+![conda-pytorch-specification](/img/how-tos/pytorch-conda-specification.png)
+
+Add any other packages you need and click `Create` to build the environment.
+
+Also see the FAQ answer to [Why doesn't my code recognize the GPU(s) on Nebari?][why gpus not recognized]
+
+[pytorch installation matrix]: https://pytorch.org/get-started
+[why gpus not recognized]: faq#why-doesnt-my-code-recognize-the-gpus-on-nebari
```
Comment (pavithraes) [inline docs/docs/how-tos/pytorch-best-practices.md:33]:
```suggestion
[why gpus not recognized]: ../faq#why-doesnt-my-code-recognize-the-gpus-on-nebari
```

This was the bit causing the test failures. :)

File: docs/docs/faq.md [hunk 2]
```diff
@@ -165,3 +165,4 @@ If you have potential solutions or can help us move forward with updates to the
 
 [dask-tutorial]: tutorials/using_dask.md
 [selecting a profile]: https://www.nebari.dev/docs/how-tos/login-keycloak#3-selecting-a-profile
+[pytorch best practices]: https://www.nebari.dev/docs/how-tos/pytorch-best-practices
```
Comment (pavithraes) [inline docs/docs/faq.md:168]:
```suggestion
[selecting a profile]: how-tos/login-keycloak#3-selecting-a-profile
[pytorch best practices]: how-tos/pytorch-best-practices
```

A nice-to-have, to clearly denote internal links. These will be easier to find & update if we do any restructuring in the future. :)

File: docs/docs/how-tos/pytorch-best-practices.md (hunk 1 above)
Comment (pavithraes) [reply docs/docs/how-tos/pytorch-best-practices.md:33]: Relevant line in the error message: https://github.com/nebari-dev/nebari-docs/actions/runs/4788922352/jobs/8516160887?pr=309#step:7:36

File: docs/docs/faq.md (hunk 2 above)
Comment (ericdatakelly) [reply docs/docs/faq.md:168]: Agreed.  I was following the lead of the keycloak link, but this is better.

**Review by pavithraes — APPROVED** [review]

**Comment by pavithraes** [comment]
Thanks, @ericdatakelly! 🌻

**Pull request merged**